testing/adios2/performance/query/TestBPQuery.cpp

```diff
@@ -31,10 +31,10 @@ void WriteXmlQuery1D(const std::string &queryFile, const std::string &ioName,
     file << "   <var name=\"" << varName << "\">" << std::endl;
     file << "      <boundingbox  start=\"5\" count=\"80\"/>" << std::endl;
     file << "       <op value=\"OR\">" << std::endl;
-    file << "         <range  compare=\"GT\" value=\"6.6\"/>" << std::endl;
+    file << "         <range  compare=\"GT\" value=\"100.6\"/>" << std::endl;
     file << "         <range  compare=\"LT\" value=\"-0.17\"/>" << std::endl;
     file << "         <op value=\"AND\">" << std::endl;
-    file << "            <range  compare=\"LT\" value=\"2.9\"/>" << std::endl;
+    file << "            <range  compare=\"LT\" value=\"11.9\"/>" << std::endl;
     file << "            <range  compare=\"GT\" value=\"2.8\"/>" << std::endl;
```

Comment:
I would not use the heat transfer data for the `Testing` files. In the examples you could use whatever you want, but when testing a function I would use some data that is easy to check correctness for. It is not clear to me why the correct answer for BP4 is `{2,1,1}`.

Reply:
@guj , can you address @anagainaru 's concerns?

Reply:
Sure, I didn't see them

Reply:
This test does not use heat transfer data. Rather it first writes a file, and then query it.